staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go

```diff
@@ -62,6 +64,13 @@ type RequestInfo struct {
 	Name string
 	// Parts are the path parts for the request, always starting with /{resource}/{name}
 	Parts []string
+
+	// FieldSelector contains the unparsed field selector from a request.  It is only present if the kube-apiserver
```

Comment:
nit: avoid kube-apiserver mentions in k8s.io/apiserver (here and below)

```suggestion
	// FieldSelector contains the unparsed field selector from a request.  It is only present if the apiserver
```